services/traction/api/db/models/base.py

```diff
@@ -3,16 +3,20 @@
 from datetime import datetime
 from typing import Optional, List
 
+import logging
 import pydantic
 from sqlalchemy.exc import DBAPIError
 from sqlmodel import Field, SQLModel
 from sqlalchemy import Column, func, text, select, desc, String
+from sqlalchemy.sql.expression import Select
 from sqlalchemy.dialects.postgresql import UUID, TIMESTAMP, VARCHAR, ARRAY
 from sqlmodel.ext.asyncio.session import AsyncSession
 
 from api.db.session import async_session
 from api.endpoints.models.v1.errors import NotFoundError
 
+logger = logging.getLogger(__name__)
```

Comment:
can we make the logger a class var, so we can determine which class/table is making the log?

Reply:
it may not work without abstract classes.  so ok to leave for now I guess... perhaps we can just be careful with the log statements to include the table name.